django_celery_beat/models.py

```diff
@@ -317,10 +318,22 @@ class Meta:
 
     @property
     def human_readable(self):
+        try:
+            c = schedules.crontab(
+                minute=self.minute,
+                hour=self.hour,
+                day_of_week=self.day_of_week,
+                day_of_month=self.day_of_month,
+                month_of_year=self.month_of_year,
+            )
+            day_of_week = cronexp(",".join(map(str, c.day_of_week)))
```

Comment:
```suggestion
            day_of_week = cronexp(",".join(str(day) for day in c.day_of_week))
```
% `ruff rule C417`

% `ruff check --select=C417 --fix django_celery_beat/models.py`

https://docs.astral.sh/ruff/rules/unnecessary-map

Reply:
Thanks!